src/main/java/me/dinowernli/grpc/prometheus/ClientMetrics.java

```diff
@@ -2,59 +2,66 @@
 
 package me.dinowernli.grpc.prometheus;
 
+import static me.dinowernli.grpc.prometheus.Common.addLabels;
+import static me.dinowernli.grpc.prometheus.Common.asArray;
+import static me.dinowernli.grpc.prometheus.Common.customLabels;
+import static me.dinowernli.grpc.prometheus.Common.metadataKeys;
+
 import java.util.ArrayList;
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
 
-import io.grpc.MethodDescriptor;
+import io.grpc.Metadata;
+import io.grpc.Metadata.Key;
 import io.grpc.Status.Code;
 import io.prometheus.client.CollectorRegistry;
 import io.prometheus.client.Counter;
 import io.prometheus.client.Histogram;
-import io.prometheus.client.SimpleCollector;
 
 /**
  * Prometheus metric definitions used for client-side monitoring of grpc services.
  */
 class ClientMetrics {
+  private static final List<String> defaultRequestLabels =
```

Comment:
Can we use Collections.unmodifiableList here?

Also, is it worth making the indentation on break consistent with the rest of the file, i.e., 4 space?

Reply:
Done